src/parser/core/CombatLogParser.js

```diff
@@ -562,13 +586,16 @@ class CombatLogParser {
     this.getModule(EventEmitter).addEventListener(...args);
   }
 
-  deepDisable(module) {
+  deepDisable(module, state = MODULE_ERROR.RESULTS) {
     console.error('Disabling', module.constructor.name);
+    if(module.active){ //prevent duplicate entries
```

Comment:
Could just early return the function even before the console.error since running deepDisable should be pointless when a module is already disabled.